_people/hyungro_lee.md

```diff
@@ -0,0 +1,36 @@
+---
+name: "Hyungro Lee"
+collection: people
+avatar: hyungro.jpg
+joined: 2018
+position: researcher
+permalink: /people/lee212
+---
```

Comment:
Would you mind changing the permalink to `first_last` name or something similar?

Reply:
changed 032d36dc9d9a04cc7a271af448c8a80cf3e93340